corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -198,8 +198,8 @@ def _bad_es_getter():
             from elasticsearch import Elasticsearch
             return Elasticsearch(
                 [{
-                    'host': 'localhost',
-                    'port': '9000',  # bad port
+                    'host': settings.ELASTICSEARCH_HOST,
+                    'port': settings.ELASTICSEARCH_PORT - 2,  # bad port
```

Comment:
Random different port than the configured one. This is not a great feature of the way this test is written because for any other random port there could be another service running that would cause an incorrect failure.